newuser/newuser.js

```diff
@@ -0,0 +1,292 @@
+const { Cipher } = require('crypto');
+const fs = require('fs')
+const path = require('path')
+const u = require('@elife/utils')
+const { ipcRenderer } = require('electron');
+const {remote} = require('electron');
+
+const Path = require('path')
+
+var pharseParam ='';
+var pharseParam2=''
+var phrase='';
+var selectrdPhraseArr=[];
+var pubkey;
+var seckey;
+var publickey;
+var secretkey;
+var elifeKeys;
+var secureKeys = '';
+var passMatch= false;
+if (document.URL.split('?')[0].includes('4.html')){
+    const params = new URLSearchParams(window.location.search)
+    for (const param of params) {
+        pharseParam=param[1].split('^^')[0];
+        pharseParam2 = param[1].split('^^')[1];
+        
+    }
+    
+    document.getElementById('generate').innerHTML = pharseParam;
```

Comment:
Why is this being set and then overwritten on line 36?

Reply:
newly constructed HTML string  i am pushing in to the existing DOM.

Reply:
You are writing two things to the DOM. The first here and the second a few lines later which will overwrite this. So what's the use of writing something only to overwrite it after?

Reply:
I wrote that line for testing initially, thanks for pointing ,I have removed that line .